variables.tf

```diff
@@ -179,3 +179,9 @@ variable "status_update_queue_message_retention_time_seconds" {
   description = "The number of seconds status_update_queue SQS retains a message in seconds. Maximum value is 14 days."
   default     = 777600 #9 days
 }
+
+variable "orca_default_multipart_chunksize_mb" {
```

Comment:
should also add this to `terraform.tfvars.example` file

Reply:
should also add this to `https://github.com/nasa/cumulus-orca/blob/develop/README.md` file

Reply:
No references to some other optional variables in these files. Should I add them all in?

Reply:
I actually added those in `temp/SharedDB` branch that you created last week. So you can only add `orca_default_multipart_chunksize_mb`